org.adempiere.base/src/org/compiere/print/ReportEngine.java

```diff
@@ -1203,7 +1178,91 @@ else if (obj instanceof PrintData)
 		return true;
 	}	//	createHTML
 
-
+	/**
+	 * Get record identifier string
+	 * @param mTable
+	 * @param tableName
+	 * @param recordID
+	 * @return String identifier
+	 */
+	private String getIdentifier(MTable mTable, String tableName, int recordID) {
+		ArrayList<MColumn> list = new ArrayList<MColumn>();
+		MClient tenant = MClient.get(getCtx(), Env.getAD_Client_ID(getCtx()));
+		// check translation table
+		MTable mTableTrl = MTable.get(getCtx(), tableName+"_Trl");
+		String tableNameTrl = "";
+		// get languages
+		String tenantLang = tenant.getAD_Language();
+		String reportLang = new MLanguage(getCtx(), getLanguageID(), null).getAD_Language();
```

Comment:
when getLanguageID() == 0, should fall backes to Language.getLoginLanguage()

Reply:
Fallback added.